convert2rhel/actions/__init__.py

```diff
@@ -226,6 +226,27 @@ def set_result(self, status=_NO_USER_VALUE, error_id=_NO_USER_VALUE, message=_NO
         if message != _NO_USER_VALUE:
             self.message = message
 
+    def add_message(self, level=_NO_USER_VALUE, id=_NO_USER_VALUE, message=_NO_USER_VALUE):
+        """
+        Helper method that adds the resulting values for level, id and message of a warning or info log message.
+
+        :param level: The level to be set.
+        :type: level: str
+        :param id: The error_id to identify the error.
+        :type id: str
+        :param message: The message to be set.
+        :type message: str | None
+        """
+        # if level != _NO_USER_VALUE:
+        #     self.level = level
+        #
+        # if id != _NO_USER_VALUE:
+        #     self.id = id
+        #
+        # if message != _NO_USER_VALUE:
+        #     self.message = message
+        pass
```

Comment:
Why is this commented out? It will not be used?

Reply:
Commented out because that method is just a placeholder until the actual add_message code is introduced from https://github.com/oamg/convert2rhel/pull/841